app/models/split_registry_snapshot.rb

```diff
@@ -0,0 +1,37 @@
+class SplitRegistrySnapshot
```

Comment:
given all the other changes, snapshot no longer makes much sense, let's just change this back to `SplitRegistry`.